frontend/src/AboutPage.js

```diff
@@ -9,9 +9,10 @@ function AboutPage() {
 
     useEffect(() => {
         getFood().then((result) => {
+            //@ts-ignore
```

Comment:
Is this comment needed? I have no idea what this does

Reply:
thanks, I'll add an explanation for it